docs/calicoctl/container.md

```diff
@@ -0,0 +1,159 @@
+
+These commands can be used to manage Calico networking for Docker containers.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl container' commands
+
+Run
+
+    calicoctl container --help
+
+to display the following help menu for the calicoctl container commands.
+
+```
+
+Usage:
+  calicoctl container <CONTAINER> ip (add|remove) <IP> [--interface=<INTERFACE>]
+  calicoctl container <CONTAINER> endpoint show
+  calicoctl container <CONTAINER> profile (append|remove|set) [<PROFILES>...]
+  calicoctl container add <CONTAINER> <IP> [--interface=<INTERFACE>]
+  calicoctl container remove <CONTAINER>
+
+Description:
+  Add or remove containers to calico networking, manage their IP addresses and profiles.
+  All these commands must be run on the host that contains the container.
+
+Options:
+  --interface=<INTERFACE>  The name to give to the interface in the container
+                           [default: eth1]
+
+```
+
+## calicoctl container commands
+
+### calicoctl container <CONTAINER> ip add <IP> 
+
+This command allows you to add an IP address to a container that is already
+using Calico networking.
+
+This command must be run on the specific Calico node that hosts the container.
+
+Command syntax:
+
+```
+calicoctl container <CONTAINER> ip add <IP> [--interface=<INTERFACE>]
+
+Parameters:
+    <CONTAINER>: The name or ID of the container
+    <IP>: The IPv4 or IPv6 address to add.
+    --interface=<INTERFACE>  The name to give to the interface in the container
+                             [default: eth1]
+    
+```
+
+Examples:
+
+```
+$ calicoctl container test-container ip add 192.10.0.3 --interface=eth1 
+```
+
+### calicoctl container <CONTAINER> ip remove <IP> 
+
+This command allows you to remove an IP address from a container that is
+using Calico networking.
+
+This command must be run as root and must be run on the specific Calico node 
+that hosts the container.
+
+Command syntax:
+
+```
+calicoctl container <CONTAINER> ip remove <IP> [--interface=<INTERFACE>]
+
+    <INTERFACE>: The name to give to the interface in the container
+                 [default: eth1]
+    <CONTAINER>: The name or ID of the container
+    <IP>: The IPv4 or IPv6 address to add.
+```
+
+Examples:
+
+```
+$ calicoctl container test-container ip remove 192.10.0.3 --interface=eth1 
+```
+
+### calicoctl container <CONTAINER> endpoint-id show
+
+This command allows you to view the IDs of the endpoint associated with
+a container.  The endpoint ID is used by the 
+[`calicoctl endpoint`](endpoint.md) commands for manipulating and viewing
+endpoint configuration.
+
+This command must be run on the specific Calico node that hosts the container.
+
+Command syntax:
+
+```
+calicoctl container <CONTAINER> endpoint-id show
+
+    <CONTAINER>: The name or ID of the container
+```
+
+Examples:
+
+```
+$ calicoctl container test-container endpoint-id show
```

Comment:
this doesn't exist anymore.

Probably worth adding something to the autogeneration script to check if the help text is out of date.